apps/nextjs/src/components/AppComponents/Chat/chat-lessonPlanDisplay.tsx

```diff
@@ -62,6 +65,24 @@ const organiseSections = [
   { trigger: "additionalMaterials", dependants: ["additionalMaterials"] },
 ];
 
+function getSectionsToDisplay(
```

Comment:
Pull out this logic so we know which sections will be available. I'm not convinced on this - won't this mean we sometimes see empty sections?